pkg/policy/rules.go

```diff
@@ -381,16 +381,14 @@ func (rules ruleSlice) updateEndpointsCaches(ep Endpoint, epIDSet *IDSet) bool {
 	return false
 }
 
-func (rules ruleSlice) refreshRulesForEndpoint(ep Endpoint) {
+func (rules ruleSlice) refreshRulesCache(ep Endpoint) {
 	if ep == nil {
 		return
 	}
 
-	id := ep.GetID16()
```

Comment:
does the `Endpoint` interface need `GetID16` as a required function anymore?

Reply:
It's still used to collect a group of endpoint IDs that need regenerating based upon policy changes---see `UpdateRulesEndpointsCaches()`.